templates/jvb-statefulset.yaml

```diff
@@ -1,3 +1,11 @@
+{{- $memoryLimit := .Values.resources.limits.memory -}}
+{{- $value := regexFind "\\d+" $memoryLimit -}}
+{{- $heapSize := "" -}}
+{{- if hasSuffix "Mi" $memoryLimit -}}
+  {{- $heapSize = mul 0.7 (int $value) | int -}}
+{{- else if hasSuffix "Gi" $memoryLimit -}}
+  {{- $heapSize = mul 0.95 (mul 1024 (int $value)) | int -}}
+{{- end -}}
```

Comment:
This is now getting evaluated outside of `{{- if ne (merge $.Values dict | dig "jvb" "resources" "limits" "memory" "") "" }}`. Is this safe to run if there's no resources/no limits/no memory limit set?